### PR TITLE
Move to bank sample

### PR DESCRIPTION
**Pull request by asnewman**
This PR moves the demo from Dittobnb to DittoPay and uses Vite instead of create-react-app.

Test plan:
- [x] Follow readme instructions to verify setup is easy and clear
- [x] Verify text is being served by the json files
- [x] Verify that the variables are working correctly
- [x] Verify switching to the Portuguese variant works correctly
- [x] Verify the app's flow works correctly

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,47 +1,33 @@
...
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject",
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "preview": "vite preview",
     "sync": "cd src && ditto-cli pull"
```
Comment (azjgard) [inline package.json:11]: Similar to our own internal conventions, I think an even better name for this command would be `ditto:pull` (and that is what @kadenbarlow has done in the Android repo: https://github.com/dittowords/ditto-android-demo/pull/1/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R11)

File: public/vite.svg
Comment (azjgard) [inline public/vite.svg:1]: maybe we replace this with a ditto icon?

File: src/assets/react.svg
Comment (azjgard) [inline src/assets/react.svg:1]: remove this?

**Review by azjgard — APPROVED** [review]
Left a couple of minor comments and pushed a commit here: https://github.com/dittowords/ditto-react-demo/pull/10/commits/263f28acf5da5583f48ff3b238a6216032d0fd41. Ran through testing and reviewed all of the code and things look good!

File: README.md [hunk 2]
```diff
@@ -1,79 +1,66 @@
...
-### 🖋 Making an example edit
+## 3. Syncing Edits ✍️
 
-1. Sign in to your Ditto account
-2. Navigate to your [component library](https://beta.dittowords.com/components/all)
-3. In the search input near the top of the screen, search for the text "Travel the world with your family"
-4. Select the component and, using the detail panel on the right side of the screen, change the text to "Go anywhere" and click **Save Edits**.
-5. In your terminal, navigate to the root of your cloned `ditto-demo` instance
-6. Run `yarn sync`
+1. Make edits to DittoPay text in Ditto ([link](https://app.dittowords.com/projects/sample)). You can make edits to the components connected in the DittoPay sample project itself, or to the sample components directly!
```
Comment (jaerod95) [inline README.md:56]: should this b the link on the ditto pice of text instead of link?

File: README.md [hunk 3]
```diff
@@ -1,79 +1,66 @@
-# Dittobnb
+# DittoPay — React Demo App
 
-Dittbnb is a small web app designed to serve as an example of how Ditto's developer tools can be integrated into a codebase.
+DittoPay is a small React app that corresponds with the sample project in your workspace. With this sample app, you can set up our [developer integrations ](https://developer.dittowords.com) to sync text edits in just a few minutes.
```
Comment (jaerod95) [inline README.md:3]: seems to be an extra space here
```suggestion
DittoPay is a small React app that corresponds with the sample project in your workspace. With this sample app, you can set up our [developer integrations](https://developer.dittowords.com) to sync text edits in just a few minutes.
```

**Pull request merged**